packages/happtiq_commons_google_cloud/src/happtiq_commons_google_cloud/gcs_api_service.py

```diff
@@ -39,7 +39,8 @@ def upload_file(self, file_path: bytes,  bucket_name: str, destination: str, con
         return f"gs://{bucket_name}/{destination}"
     
     def list_files_with_prefix(self, bucket_name: str, prefix: str):
-        self.logger.info(f"Listing files in bucket {bucket_name} with prefix {prefix}")
+        self.logger.debug(f"Files in bucket {bucket_name} with prefix {prefix}")
+        self.logger.info(f"found {len(file_list)} files in bucket {bucket_name} with prefix {prefix}: {file_list}")
```

Comment:
this is completely wrong. I didnt say to change this log

I said t o change other log

Reply:
please recheck it again.. I corrected it in the last commit

Reply:
now correct